PowerEditor/src/ScitillaComponent/AutoCompletion.cpp

```diff
@@ -123,14 +147,23 @@ bool AutoCompletion::showApiAndWordComplete()
 void AutoCompletion::getWordArray(vector<generic_string> & wordArray, TCHAR *beginChars)
 {
 	const size_t bufSize = 256;
-
+	if (isAllDigits(beginChars))

```

Comment:
Forcing this upon a user isn't a good idea. They may well want numbers to be autocompleted. I have PR #2378 that does this exact same thing but allows the user to enable it or not.